typings/restify-validator/restify-validator.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for adding validator to restify
```

Comment:
I think we should put this typing file alongside with the `express-validator` since it is not a `restify-validator`.